src/debugAdapter/goDebug.ts

```diff
@@ -869,26 +869,34 @@ class GoDebugSession extends LoggingDebugSession {
 				breakpointIn.loadLocals = this.delve.loadConfig;
 				breakpointIn.cond = breakpoint.condition;
 				return this.delve.callPromise('CreateBreakpoint', [this.delve.isApiV1 ? breakpointIn : { Breakpoint: breakpointIn }]).then(null, err => {
+					if (err.toString().startsWith('Breakpoint exists at')) {
```

Comment:
Is there no other way to check for this error other than checking the error message this way? Maybe an error code or error name?
Error messages can change in the future, so I am a little weary of depending on them this way.